### PR TITLE
LocationPage now requires BIRTHDATE instead of AGE

### PR DESCRIPTION
**Pull request by alfonsrv**
ImpfterminService site changed.  

Not familiar with kotlin config files, so you might have to check if it adds up.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,4 @@
+doc/* linguist-documentation
+src/test/* linguist-vendored
```
Comment (TobseF) [inline .gitattributes:2]: Sure you also wanted to add this file?

File: src/main/resources/config.properties [hunk 2]
```diff
@@ -4,8 +4,8 @@ state = Baden-Württemberg
 # Comma separated list of locations. Optional, if you already have a placement code just add it in square brackets after the place. Furthermore you can optionally specify the related server code next to the placement code in parentheses (can be found in the URL e.g. "001-iz.impfterminservice.de -> server code == 001).
 locations = 69124 Heidelberg[XXXX-XXXX-XXXX](XXX),76137 Karlsruhe
 
-# Your age. Used in age verification field.
-personAge = 42
+# Your age. Used in birthdate verification field. DD.MM.YYYY
+personBirthdate = 31.12.1994
```
Comment (TobseF) [inline src/main/resources/config.properties:8]: You also have to change name and type in the `Config` class (`kotlin/de/tfr/impf/config/Config.kt`):

```kotlin
val personAge: Int by lazyIntProperty()
// replace by
val personBirthdate : String by lazyProperty()
```
In addition, you should also replace the `personAge` in the `README`.

File: .gitattributes (hunk 1 above)
Comment (alfonsrv) [reply .gitattributes:2]: Do you want your repo to be a HTML repo?

Comment (TobseF) [reply .gitattributes:2]: Didn't know that, but sounds great 👍

**Comment by TobseF** [comment]
Already done by https://github.com/TobseF/impf-bot/commit/e34d36e3ebd773c59360b86287be1b55b3ad4b41.
Thanks for reporting and the changes.

**Pull request closed without merging**